### ISSUE TITLE
Can't pass both ClusterID and ClusterObject to Update-DatabricksCluster

### ISSUE THREAD

**Issue by lorenzo-romanelli** [gbrueckl/Databricks.API.PowerShell#38]
When invoking the cmdlet `Update-DatabricksCluster` passing both the ClusterID and the ClusterObject, the execution fails:

```powershell
$clusterToUpdate = $clusterJsonObject | ConvertFrom-Json
Update-DatabricksCluster -ClusterID "xxxxxxxxxxxxxxxxxxxxxxxx" -ClusterObject $clusterToUpdate
```
with this error
```
Parameter 'ClusterID' cannot be specified in parameter set 'ClusterObject'.
```

It seems that it is not possible to pass both parameters to the cmdlet (this behaviour was not encountered at least up to 1.2.1.0).

DatabricksPS Version: 1.8.0.1

**Comment by gbrueckl** [comment]
hey Lorenzo,

I guess you want to -ClusterID to overwrite the value of the -ClusterObject in this case - right?

**Comment by lorenzo-romanelli** [comment]
hi Gerhard! thanks for the quick reply :)

What I would expect is that the `-ClusterID` (existing cluster) is updated according to the specifications in the `-ClusterObject` definition. Not sure what should happen if I also specify a cluster ID in the `-ClusterObject` - maybe raise an error if the ID is reported twice?

**Comment by gbrueckl** [comment]
it used to be that the `-ClusterID` did overwrite the cluster_id provided via the `-ClusterObject` which is also the case for all other attributes.
however, this setup was quite complicated after switching to a dynamic parameter for `-ClusterID` 
let me see if I can get the `-ClusterID` to work like all the other parameters and overwrite the value if `-ClusterObject` is also specified

**Comment by gbrueckl** [comment]
ok I fixed it in v1.8.1.0
you can now specify both parameters at the same time which would re-configure the cluster specified via `-ClusterID` with the configs specified in `-ClusterObject`
it will throw a warning though that the `cluster_id` was specified twice (in case `-ClusterObject` also has the property `cluster_id` defined)

let me know if this works for you and close the ticket if yes

**Comment by lorenzo-romanelli** [comment]
worked like a charm, thank you!
